client/stylesheets/_toCat.scss

```diff
@@ -2025,18 +2045,35 @@ i:hover {
     margin: 3rem 0 1rem 0;
 }
 
+.helpCenterThree {
+    border-top: 1px solid #404040;
+    text-align: left !important;
+}
+
+.helpThree > li {
+    width: 28%;
+    display: inline-block;
+}
+
+.helpThree li ul li h3 {
+    margin: 5px 0 4px 0;
+}
+
+.helpThree li ul li p {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 5<br>Selector `helpThree` should be written in lowercase with hyphens